server/services/interventionsService.ts

```diff
@@ -52,6 +52,15 @@ export interface SentReferral {
   endOfServiceReport: EndOfServiceReport | null
 }
 
+export interface EndedReferral {
+  id: string
+  referenceNumber: string
+  referral: ReferralFields
+  endedAt: string
+  endedBy: AuthUser
+  cancellationReason: string
```

Comment:
Are these arbitrary, or are there a list of them somewhere? e.g. is it worth specifying a stronger type/enum?

Reply:
there is a list but it's subject to change and not part of the contract as such i don't think.

they are three letter codes. i could define a type that's three letters explicitly?